pytorch_lightning/profiler/xla.py

```diff
@@ -63,8 +36,13 @@ class XLAProfiler(BaseProfiler):
     }
 
     def __init__(self, port: int = 9012) -> None:
-        """This Profiler will help you debug and optimize training workload performance for your models using Cloud
-        TPU performance tools."""
+        """XLA Profiler will help you debug and optimize training workload performance for your models using Cloud
+        TPU performance tools.
+
+        Args:
+            port : the port to start the profiler server on. An exception is
+                raised if the provided port is invalid or busy. Default: ``9012``.
```

Comment:
```suggestion
            port : the port to start the profiler server on. An exception is
                raised if the provided port is invalid or busy.
```
defaults are added by sphinx